src/components/comparison-params-panel/comparison-params-panel.tsx

```diff
@@ -0,0 +1,151 @@
+import { useState } from "react";
+import styled from "styled-components";
+import { getCurrentLayoutProperty, useAppLayout } from "../../utils/layout";
+import { Theme } from "../../types";
+import { ToggleSwitch } from "../toogle-switch/toggle-switch";
+
+type ComparisonParamsProps = {
+  id: string;
+  onClose: () => void;
+};
+
+type LayoutProps = {
+  layout: string;
+};
+
+const Container = styled.div<LayoutProps>`
```

Comment:
You have a lot of copy-pasted components here. Create `comparison` folder. Put reusable components in `common.tsx` file. Use components in both panels